Jenkinsfile

```diff
@@ -29,7 +29,10 @@ pipeline {
       }
       stage('Build Release') {
         when {
-          branch 'master'
+          allOf {
+              branch 'master'
+              not { changelog '^release v[0-9\.]+$' }
```

Comment:
@pmuir. I am wondering whether something like this could stop the endless loop. If the changelog contains the 'release' commit don't do another release build.

It would be nicer to be able to configure Prow to ignore a given commit, but this might work. WDYT?

Not sure really how to test this unless to actually merge and see what happens, but without knowing how to kill the build in case of an emergency I don't dare.

Reply:
Hmm, I don't love it.

I can show you how to kill the build tomorrow. Or just create a test project in your own cluster...